aws/resource_aws_glue_job.go

```diff
@@ -107,6 +107,7 @@ func resourceAwsGlueJob() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"tags": tagsSchema(),
```

Comment:
I'm also not sure if this is the correct tag function or if it should be `tagsSchemaForceNew()`

Reply:
The resource does not need to be replaced to update its resource tags, so `tagsSchema()` is appropriate. 👍